code/modules/surgery/implant_removal.dm

```diff
@@ -1,81 +1,199 @@
 /datum/surgery/implant_removal
-	name = "Implant removal"
+	name = "Implant extraction/destruction"
 	possible_locs = list(BODY_ZONE_CHEST)
 	steps = list(
 		/datum/surgery_step/incise,
 		/datum/surgery_step/clamp_bleeders,
 		/datum/surgery_step/retract_skin,
-		/datum/surgery_step/extract_implant,
+		/datum/surgery_step/search_for_implants,
+		/datum/surgery_step/destroy_extract_implant,
 		/datum/surgery_step/close,
 	)
+	var/list/discovered_implants = list()
 
-//extract implant
-/datum/surgery_step/extract_implant
-	name = "extract implant (hemostat)"
+/datum/surgery_step/search_for_implants
+	name = "search for implants (hand/t-ray)"
+	accept_hand = TRUE
 	implements = list(
-		TOOL_HEMOSTAT = 100,
-		TOOL_CROWBAR = 65,
-		/obj/item/kitchen/fork = 35)
-	time = 64
-	success_sound = 'sound/surgery/hemostat1.ogg'
-	var/obj/item/implant/implant
+		/obj/item/t_scanner = 100,
+	)
+	time = 12.8 SECONDS
```

Comment:
This seems like a _very_ long time, especially for a repeatable step with a reasonably high failure chance which may need to be performed multiple times.
Making implants more deniable is a good idea but I think that even people it is being done to won't appreciate being on the table for the additional _minutes_ it takes to have a reasonable chance of discovering an implant using the best possible tool.

Reply:
I figured that the numbers would need tweaking, do you have better suggestions for detection prob/time for step?

Reply:
I'm thinking 5 seconds would be good, but Jacq may have something else in mind.